sunpy/net/base_client.py

```diff
@@ -203,7 +203,7 @@ def _print_client(client, html=False):
     if html:
         lines = [f"<p>{line}</p>" for line in lines]
     lines.extend(t.pformat_all(show_dtype=False, max_width=width, align="<", html=html))
-    return '\n'.join(lines)
+    return '\n'.join(lines[slice(0, visible_entries)])
```

Comment:
I think if we truncate this output (i.e. `visible_entries is not None`) then we should append `...` to the output or similar so that it's obvious it's been truncated.